### PR TITLE
[e2e Test] WIP e2e batch transactions

### PR DESCRIPTION
**Pull request by francovenica**
## What it solves

Add e2e for the batch tx feature

## How this PR fixes it
Tests:
Empty state
Add to batch button present in the tx form
Adding 2 tx in a batch
Swapping tx order
Checking the order in the tx form
Delete a tx from the batch

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: cypress/e2e/smoke/batch_tx.cy.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    cy.contains('Accept selection').click()
+  })
+
+  it('Should open an empty batch list', () => {
+    cy.get(BATCH_TX_TOPBAR).should('be.visible').click()
+    cy.contains('Batched transactions').should('be.visible')
+    cy.contains('Add an initial transaction to the batch')
+    cy.get(ADD_NEW_TX_BATCH).click()
+  })
+  it('Should see the add batch button in a tx form', () => {
```
Comment (usame-algan) [inline cypress/e2e/smoke/batch_tx.cy.js:23]: nit: An empty line between each block makes it easier to read

File: cypress/e2e/smoke/batch_tx.cy.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    addToBatch(EOA, currentNonce, funds_first_tx, true)
+  })
+  it('Should see the tx being added to the batch', () => {
+    cy.contains('Transaction is added to batch').should('be.visible')
+    //Counter The batch button in the header
```
Comment (usame-algan) [inline cypress/e2e/smoke/batch_tx.cy.js:28]:
```suggestion
    // The batch button in the header shows the transaction count
```

File: cypress/e2e/smoke/batch_tx.cy.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  it.skip('Should confirm the batch and see 2 tx in the form', () => {
+    cy.contains('Confirm batch').click()
+    //This one is hard since the list of tx to execute are divs on divs, and not an ul that I can count or compare with anything
+    //and there are no ids or classes to use
+  }) //TODO'
+  it('Should remove a tx from the batch', () => {
+    cy.contains('Batched transactions').should('be.visible')
+    cy.contains(funds_first_tx).parents('ul li').find('[title="Delete transaction"]').eq(0).click()
+    cy.contains(funds_first_tx).should('not.exist')
+  }) //TODO
```
Comment (usame-algan) [inline cypress/e2e/smoke/batch_tx.cy.js:57]: Is this TODO still needed?

**Comment by francovenica** [comment]
Added the suggestions. 
Kept the test of the swap with a skip so we can revisit later
Added the test that validates the order of the tx in the form

**Review by usame-algan — APPROVED** [review]
Looks good 👍

**Pull request merged**